core/dbt/tests/util.py

```diff
@@ -94,6 +99,16 @@ def run_dbt_and_capture(args: List[str] = None, expect_pass=True):
     return res, stdout
 
 
+# Use this if you need to capture the standard out in a test
```

Comment:
Why duplicate the 'run_dbt_and_capture' functions? What does this do different?

Reply:
`dbt debug` contains `print()` statements which weren't showing up in the output for `run_dbt_and_capture`.

In my hands-on experiments, `run_dbt_and_capture` doesn't actually capture standard out -- I think it captures content streamed to the logs.

So I created a variant that includes the output of `print()` statements.

Reply:
We don't need two different variations of this function. Looking at the original one, it looks like it certainly intended to capture the stdout logs, if you look at the 'capture_stdout_logs' in core/dbt/events/functions.py. 

If there's something that it's missing, we should fix it rather than create an almost duplicate. What do you mean by "print()" statements? We shouldn't have any actual print statements to capture, should we?

Reply:
I used `git grep "print(" ./` to find instances of `print()`.

`dbt debug` uses `print()` statements. There are a some other places too, but I didn't examine them in detail.

Do you know how to modify `capture_stdout_logs` to capture `print()` statements? Or some other alternative to avoid functions that are near duplicates? I saw how `capture_stdout_logs` captures logging, but didn't see any obvious way for it to capture standard out also.